modules/apps/object/object-rest-test/src/testFunctional/tests/ManageSystemObjectRelationshipWhenRelatedCustomObjectIsDeactivated.testcase

```diff
@@ -0,0 +1,248 @@
+			UserAccountAPI.setUpGlobalUserAccountIds(
+				alternateName1 = "user1",
+				alternateName2 = "user2",
+				emailAddress1 = "user1@liferay.com",
+				emailAddress2 = "user2@liferay.com",
+				familyName1 = "userfn1",
+				familyName2 = "userfn2",
+				givenName1 = "usergn1",
+				givenName2 = "usergn2");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		for (var objectLabel : list "foundation,secondFoundation") {
+			ObjectAdmin.openObjectAdmin();
+
+			CreateObject.activeCustomObject(objectLabel = ${objectLabel});
+		}
+
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		JSONUser.tearDownNonAdminUsers();
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 4
+	test CannotGetManyToManyRelatedEntryDetailsWhenCustomObjectIsDeactivated {
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "ManageSystemObjectRelationshipWhenRelatedCustomObjectIsDeactivated#CannotGetManyToManyRelatedEntryDetailsWhenCustomObjectIsDeactivated";
+
+		task ("And Given foundation entries related with userAccount through putUserAccountUsersFoundations") {
+			UserAccountAPI.relateObjectEntries(
+				customObjectId = ${staticObjectEntryId1},
+				relationshipName = "usersFoundations",
+				userAccountId = ${staticUserAccountId1});
+
+			UserAccountAPI.relateObjectEntries(
+				customObjectId = ${staticObjectEntryId2},
+				relationshipName = "usersFoundations",
+				userAccountId = ${staticUserAccountId2});
+		}
+
+		task ("When I deactivate the custom object") {
+			ObjectAdmin.openObjectAdmin();
+
+			CreateObject.inactiveCustomObject(objectLabel = "foundation");
+		}
+
+		task ("Then I receive 404 from the get request getUserAccount${relationshipName}Page") {
+			var response = UserAccountAPI.getRelationshipByUserAccountId(
+				relationshipName = "usersFoundations",
+				userAccountId = ${staticUserAccountId1});
+
+			ObjectDefinitionAPI.assertStatusInResponse(
+				expectedValue = "NOT_FOUND",
+				responseToParse = ${response});
+		}
+
+		task ("And Then the endpoint getUserAccount${relationshipName}Page is not visible in the OpenAPI") {
+			APIExplorer.navigateToOpenAPI(
+				api = "headless-admin-user",
+				version = "v1.0");
+
+			AssertElementNotPresent(
+				locator1 = "OpenAPI#API_METHOD",
+				method = "getUserAccountUsersFoundationsPage",
+				service = "UserAccount",
+				value1 = "getUserAccountUsersFoundationsPage");
+		}
+	}
+
+	@priority = 3
+	test CannotGetManyToManyRelationshipDetailsWhenCustomObjectIsDeactivated {
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "ManageSystemObjectRelationshipWhenRelatedCustomObjectIsDeactivated#CannotGetManyToManyRelationshipDetailsWhenCustomObjectIsDeactivated";
+
+		task ("When I deactivate the custom object") {
+			ObjectAdmin.openObjectAdmin();
+
+			CreateObject.inactiveCustomObject(objectLabel = "foundation");
```

Comment:
This is great to using UI as a workaround, do we have a ticket for inactive custom object doesn't work via API?

Reply:
I think not, and I sent a question about "Can not set active=false by the manyToMany relationship via API" in the [slack channel](https://liferay.slack.com/archives/C026NQPV6TV/p1679312385674979)